metadataserver/server.go

```diff
@@ -235,6 +235,8 @@ func (ms *MetadataServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		"uri":           r.RequestURI,
 		"path":          r.URL.Path,
 		"token-present": len(r.Header.Get("X-aws-ec2-metadata-token")) > 0,
+		"referer":       r.Header.Get("Referer"),
+		"taskid":        ms.titusTaskInstanceID,
```

Comment:
So, task ID is already present in the logs in the systemd unit name. Do you want duplication?

Reply:
yeah, i saw that.  it's a pain to get out of there in elastic or bpd.  requires tokenizing the unit name and what not.

Reply:
I know that it's a pain to get from elastic, but it shouldn't be from BDP. Let me know if you need help with that.